tests/integration/update_cluster/additional_cidr/cloudformation.json.extracted.yaml

```diff
@@ -133,6 +133,12 @@ Resources.AWSAutoScalingLaunchConfigurationmasterustest1bmastersadditionalcidrex
 
   cat > cluster_spec.yaml << '__EOF_CLUSTER_SPEC'
   cloudConfig: null
+  containerRuntime: docker
+  containerd:
```

Comment:
I wonder if seeing `docker` and `containerd` is going to cause user confusion

Reply:
Depends a lot on users and how we present this in the release notes and docs.
My guess is that most users either know that Kubernetes runs on Docker + containerd, or think Kops does something magic and Kubernetes just runs. Those confused will ask in `#kops-users` and we should direct them to the release notes and docs.